assets/scss/_accordion.scss

```diff
@@ -0,0 +1,37 @@
+//
+// Animation values taken from https://github.com/google/material-design-lite/blob/mdl-1.x/src/_variables.scss
+// Maybe we can bring this into `swarm-constants` or `swarm-animation`?
+//
+
+$A_curve-fast-out-slow-in: cubic-bezier(0.4, 0, 0.2, 1);
+$A_curve-default: $A_curve-fast-out-slow-in;
+$A_duration: 0.2s;
+
+.accordionPanel-animator {
+	overflow: hidden;
+	transition: height $A_duration $A_curve-default;
+}
+
```

Comment:
@unjust - 100% agreed that this stuff should live in `swarm-animation`, but I didn't think it was ready to be used yet.
If we're ready to use `swarm-animation` - let me know and we'll do a follow-up to get that into `meetup-web-components`